sbpy/units/core.py

```diff
@@ -125,3 +128,148 @@ def spectral_density_vega(wfb):
         (flam0.unit, VEGA, lambda x: x / flam0.value,
          lambda x: x * flam0.value)
     ]
+
+
+def magnitude_reflectance(xsec, wfb=None, M_sun=None):
```

Comment:
For example, if I want to convert Deep Impact spectra to reflectance, I use a function named `magnitude_reflectance`, which does not seem right.

Reply:
You are right here, but in the future I will define some equivalencies to support conversion between various reflectance units (bidirectional, RADF, REFF, BRDF, etc).  I think it's more appropriate to call that equivalency `reflectance`.  What about we call this one `flux_reflectance`?

Reply:
I looked at http://docs.astropy.org/en/stable/units/equivalencies.html for guidance, and equivalency names are more about the concept rather than the two items being converted.

For all these reflectance units, can they not just be returned by equivalency, like the way that `u.spectral_density` returns 16 different conversions?  Why separate functions for each one?

Reply:
Good point.  We can do that.

Reply:
I merged the two conversions into one single equivalence function `reflectance`, and will use this same function to support conversions between different reflectance units in the future.

Reply:
Sounds good.